src/index.html

```diff
@@ -1,18 +1,59 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <title>Stars</title>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
-    <title>Stars</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
In this case, you have duplicate element `<div class="stars__star"></div>`. For cases with duplication, it is better to use a list, not div.